### PR TITLE
update cluster operator doc

### LINKED ISSUE #131: Section<Install cluster operator>lacks infos on  <your_custom_values>.yaml
**What would you like to be added**:

In https://github.com/kurator-dev/kurator/blob/main/docs/content/en/docs/Cluster%20operator/quickstart.md#install-cluster-operator,<your_custom_values>.yaml comes out of nowhere
**Why is this needed**:
,confused about  file "<your_custom_values>.yaml". Cannot find the purpose of this file and what inside it.
@zirain

### PR DESCRIPTION
**Pull request by zirain**
Signed-off-by: hejianpeng <hejianpeng2@huawei.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #131 #130 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
update cluster operator doc
```

### CODE REVIEW

File: docs/content/en/docs/Cluster operator/quickstart.md [hunk 1]
```diff
@@ -52,13 +52,33 @@ Load image to kind cluster:
 kind load docker-image ghcr.io/kurator-dev/cluster-operator:latest --name kurator
 ```
 
+## Prepare AWS Credentials
+
+Download the latest binary of clusterawsadm from the [AWS provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/tag/v2.0.0). The clusterawsadm command line utility assists with identity and access management (IAM) for [Cluster API Provider AWS](https://cluster-api-aws.sigs.k8s.io/).
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/Cluster operator/quickstart.md:57]: Install?

File: docs/content/en/docs/Cluster operator/quickstart.md [hunk 2]
```diff
@@ -52,13 +52,33 @@ Load image to kind cluster:
...
+
+```console
+export AWS_REGION=us-east-1 # This is used to help encode your environment variables
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/Cluster operator/quickstart.md:62]: It is also worth mentioning how to get them

File: docs/content/en/docs/Cluster operator/quickstart.md (hunk 1 above)
Comment (zirain) [reply docs/content/en/docs/Cluster operator/quickstart.md:57]: updated

File: docs/content/en/docs/Cluster operator/quickstart.md (hunk 2 above)
Comment (zirain) [reply docs/content/en/docs/Cluster operator/quickstart.md:72]: updated

**Review by hzxuzhonghu — APPROVED** [review]

**Review by hzxuzhonghu — COMMENTED** [review]
/lgtm

**Pull request merged**